bundles/org.openhab.core.io.rest/src/main/java/org/openhab/core/io/rest/Stream2JSONInputStream.java

```diff
@@ -49,10 +51,6 @@
      * @throws IllegalArgumentException in case the source is null.
```

Comment:
If it is annotated @NonNull it can't (shouldn't) throw `IllegalArgumentException` if the parameter is null.

Reply:
I've fixed all the comments! :+1: